xrpl/model/transactions/validations_commons.go

```diff
@@ -81,14 +81,15 @@ func ValidateBaseTransaction(tx FlatTransaction) error {
 }
 
 func ValidateRequiredField(tx FlatTransaction, field string, checkValidity func(interface{}) bool) error {
```

Comment:
Should return `(boolean, error)`:

* boolean returns if the field is valid or not.
* error returns any missbehavior within the function execution